library/prime/new_spec.rb

```diff
@@ -1,22 +1,17 @@
 require File.expand_path('../../../spec_helper', __FILE__)
 require 'prime'
 
-describe "Prime.new" do
-  it "returns a new object representing the set of prime numbers" do
-    Prime.new.should be_kind_of(Prime)
+describe "Prime.instance" do
+  it "returns an object representing the set of prime numbers" do
+    Prime.instance.should be_kind_of(Prime)
   end
 
-  it "returns a object with obsolete featrues" do
-    Prime.new.should be_kind_of(Prime::OldCompatibility)
-    Prime.new.should respond_to(:succ)
-    Prime.new.should respond_to(:next)
+  it "does not respond to obsolete features" do
+    Prime.instance.should_not respond_to(:succ)
+    Prime.instance.should_not respond_to(:next)
   end
-
-  it "complains that the method is obsolete" do
-    lambda { Prime.new }.should complain(/obsolete.*use.*Prime::instance/)
-  end
-
+  
   it "raises a ArgumentError when is called with some arguments" do
-    lambda { Prime.new(1) }.should raise_error(ArgumentError)
+    lambda { Prime.instance(1) }.should raise_error(ArgumentError)
   end
```

Comment:
All the cases in new_spec are covered in instance_spec.rb, so we can remove the file without merging.